cmd/renterd/node.go

```diff
@@ -19,6 +20,8 @@ import (
 	"golang.org/x/crypto/blake2b"
 )
 
+const defaultAutopilotLoopInterval = time.Minute
```

Comment:
should be part of the autopilot config, right?

Reply:
Yeah thought about that, figured it was a bit early to make that configurable but I'll move it.